Assets/Scripts/PlayerMovement.cs

```diff
@@ -7,8 +7,8 @@ public class PlayerMovement : MonoBehaviour
     private float moveInput; // detects left or right key when pressed
     private  Rigidbody2D ballRigidBody; // variable for rigid body (private to only access it from the script)
     private bool grounded; // ground variable changes from true or false when object enters or exits collision with the ground
```

Comment:
you are not using `grounded` anyomore. before changeing, just look at the code and see where it gets used (Hint: nowhere).
There is a warning if you hover your mouse over the variable and you can also see the variable is greyed out. Read what it says before changing anything.